core/src/main/java/org/javarosa/core/model/FormDef.java

```diff
@@ -1333,13 +1333,14 @@ public void populateDynamicChoices(ItemsetBinding itemset, TreeReference curQRef
                 new EvaluationContext(exprEvalContext, itemset.contextRef.contextualize(curQRef)));
 
         if (matches == null) {
-            throw new XPathException("Could not find references depended on by " + itemset.nodesetRef.getInstanceName());
+            throw new XPathException("No items found at '" + itemset.nodesetRef +
```

Comment:
This error can pop up from non-lookup table select questions, so I wanted to leave it sorta vague. Is there a way to make `Make sure the 'someInstance' lookup table is available` clearer?